File: app/src/test/java/bogomolov/aa/fitrack/core/model/TrackTest.java
package bogomolov.aa.fitrack.core.model;

import org.junit.Test;

import java.util.Arrays;

import static org.junit.Assert.*;

public class TrackTest {

    @Test
    public void sumTracks() {
        Track track1 = new Track();
        track1.setDistance(100);
        Track track2 = new Track();
        track2.setDistance(100);
        Track sumTrack = Track.Companion.sumTracks(Arrays.asList(track1, track2));
        assertEquals(sumTrack.getDistance(), 200,0);
    }
}

File: app/src/test/java/bogomolov/aa/fitrack/core/model/TrackTest.kt
package bogomolov.aa.fitrack.core.model

import org.junit.Test

import java.util.Arrays

import org.junit.Assert.*

class TrackTest {

    @Test
    fun sumTracks() {
        val track1 = Track()
        track1.distance = 100.0
        val track2 = Track()
        track2.distance = 100.0
        val sumTrack = Track.sumTracks(Arrays.asList(track1, track2))
        assertEquals(sumTrack.distance, 200.0, 0.0)
    }
}